tests/TestCase.php

```diff
@@ -6,7 +6,7 @@
 use Sourcetoad\Logger\LoggerFacade;
 use Sourcetoad\Logger\LoggerServiceProvider;
 
-class TestCase extends \PHPUnit\Framework\TestCase
+class TestCase extends \Orchestra\Testbench\TestCase
```

Comment:
Could this be moved to an import with `use`?

Reply:
Yeah I probably should've done that.

Reply:
Done ✅